app/src/main/java/com/codesoom/assignment/errors/WrongPasswordException.java

```diff
@@ -0,0 +1,10 @@
+package com.codesoom.assignment.errors;
+
+import com.codesoom.assignment.dto.UserLoginData;
+
+public class WrongPasswordException extends RuntimeException{
+    public WrongPasswordException(UserLoginData loginData){
+        super("잘못된 비밀번호입니다. : " +loginData.getEmail());
```

Comment:
이렇게 할 수도 있겠죠. 이 예외가 발생해서 문제가 생겼을 때 디버깅해야 하는 사람은 다른 사람이 아니에요. 이걸 코딩한 나 자신일 가능성이 가장 높습니다.
```suggestion
        super("잘못된 비밀번호입니다. 로그인에 실패한 이메일은 [" + loginData.getEmail() + "] 입니다.");
```